internal/sqlchannel/engine/mongodb.go

```diff
@@ -22,8 +22,12 @@ package engine
 import (
 	"fmt"
 	"strings"
+
+	corev1 "k8s.io/api/core/v1"
 )
 
+var _ Interface = &mongodb{}
```

Comment:
this `Interface` name is too generic, perhaps, refactor to a more appropriate name

Reply:
Used `ClusterCommands` instead, as it works as a command templates provider.